koe/bot.py

```diff
@@ -800,7 +800,7 @@ async def on_message(
                         await message.channel.send("> disable", reference=message, mention_author=False)
                     return
                 elif command[0] in ["toggle", "~"]:
-                    value = not self.user_config_map[message.author.id]["enabled"]
+                    value = message.author.id not in self.enabled_user_ids_map[message.channel.id]
```

Comment:
채널 ID 키가 존재하지 않을 경우 명령이 실패합니다.

```suggestion
                    value = message.author.id not in self.enabled_user_ids_map.get(message.channel.id, [])
```